client.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Some copyrights are 2021 some are 2022 (I think all new added files), is that intentional. Will you update others later?

Reply:
That's not intentional, but I don't think it's very important. We can update the copyright headers altogether after we merge the feature branches.